pkg/ddevapp/app_compose_template.yaml

```diff
@@ -10,7 +10,7 @@ services:
         BASE_IMAGE: $DDEV_DBIMAGE
         username: '{{ .Username }}'
         uid: '{{ .UID }}'
-        gid: {{ if ne .DBType "postgres" }} {{ .GID }} {{ else }} "999" {{ end }}
```

Comment:
I just wish I could remember why this was explicitly done. There was certainly a reason.